### PR TITLE
Other cathegory for credit card number

### PR DESCRIPTION
**Pull request by babenek**

### CODE REVIEW

File: meta/798d34aa.csv [hunk 1]
```diff
@@ -215,5 +215,5 @@ Id,FileID,Domain,RepoName,FilePath,LineStart:LineEnd,GroundTruth,WithWords,Value
 130756,78e33b5c,GitHub,798d34aa,data/798d34aa/src/bz.ts,622:622,F,F,,,F,F,,,,,0.00,,F,F,F,
 130757,e3bc2547,GitHub,798d34aa,data/798d34aa/src/bk.ts,10:10,F,F,,,F,F,,,,,0.00,,F,F,F,
 130758,78e33b5c,GitHub,798d34aa,data/798d34aa/src/bz.ts,16:16,F,F,,,F,F,,,,,0.00,,F,F,F,
-131694,c679447c,GitHub,798d34aa,data/798d34aa/other/be.md,81:81,T,F,,,F,F,,,,,0,0,F,F,F,
+131701,c679447c,GitHub,798d34aa,data/798d34aa/other/be.md,81:81,T,F,,,F,F,,,,,0,0,F,F,F,Other
```
Comment (csh519) [inline meta/798d34aa.csv:218]: Why you change the `Id`?
Is it different line with 131694?

Comment (babenek) [reply meta/798d34aa.csv:218]: duplicate  with id meta/a0abd87c.csv

Comment (csh519) [reply meta/798d34aa.csv:218]: Yeah `Id` was duplicated but only for `Id`?
I wonder you've set the `Id` because of same content between those two or not.

Comment (babenek) [reply meta/798d34aa.csv:218]: The id is not used in the benchmark. Just for common identification.

**Review by csh519 — APPROVED** [review]
Thank you for fix!

LGTM 👍

**Review by iuriimet — APPROVED** [review]

**Pull request merged**